loops_lab.js

```diff
@@ -0,0 +1,222 @@
+// // ## Exercises
+// //
+// // 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+console.log('----- #1 while loop -----');
+let num = 10;
+while (num > 0) {
+  console.log(num);
+  num--;
+}
+
+console.log('----- #1 for loop -----');
+for (let num1 = 15; num1 > 0; num1--) {
+  console.log(num1);
+}
+//
+// // 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// // For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+console.log('----- #2 while loop -----');
+let max = 10;
+let counter = 0;
+while (counter <= max) {
+  if (counter % 2 === 0) { console.log(counter + ' is even');
+  }   else {
+    console.log(counter + ' is odd');
+  }
+  counter++;
+}
+//
+console.log('----- #2 for loop -----');
+let max1 = 10;
+for (let counter1 = 0; counter1 <= max1; counter1++) {
+  if (counter1 % 2 === 0) { console.log(counter1 + ' even');
+  }  else {
+    console.log(counter1 + ' odd');
+  }
+}
+//
+// // 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+// // For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
+console.log('----- #3 while loop "cannot get the numbers to ascend" -----');
+// // cannot get the numbers to ascend
+let x = 11;
+let xx = 0;
+while (xx <= x){ //&& 0 <= x)
+
+  let num2 = xx * 9;
+  console.log(xx + ' * 9 = ' + num2);
+  xx++;
+}
+//
+console.log('----- #3 for loop "cannot get the numbers to ascend" -----');
+// // cannot get the numbers to ascend
+// let xx1 = 0;
+for (let x1 = 0; x1 <= x; x1++) {
+  console.log( x1 + ' * 9 = ' + (x1*9));
+}
+
+
+// _Bonus_ think of another way to solve it.
+//         Hint
+//       Find the final number and increment the loop by 9.
+// let x = 10;
+// let num1 = [];
+// while (x >= num1*9) {
+//   console.log(x + ' * 9 = ' + num1);
+//   x--;
+// }
+
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+console.log('----- #4 -----');
+let fives = 5;
```

Comment:
What if fives become 1 instead of 5? Would it give same result? Can we somehow make this code better with use of modulo? (i.e: fives % 5 && fives %% 10). Think about implementation using modulo when you see questions like this next time.